common/src/main/scala/gu/msnotifications/Responses.scala

```diff
@@ -37,18 +39,47 @@ object XmlParser {
 
 object Responses {
 
-  implicit class RichXmlElem(xml: scala.xml.Elem) {
+  implicit class RichXmlElem(xml: Elem) {
 
     def getStrings(s: String): Seq[String] = (xml \ s).map(_.text)
 
     def getString(s: String): HubResult[String] =
       getStrings(s).headOption \/> HubParseFailed(body = xml.toString(), reason = s"Missing field $s")
 
+    def getOptionString(s: String): HubResult[Option[String]] =
```

Comment:
`get` naming convention in this class reminds me POJOs ;) I'm not a fan of that + I prefer to name all *option* accessors like 
```
def stringOption(path: String)
```